modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -25,9 +28,23 @@ import {FDSViewType} from '../FDSViews';
 import {IPickList, getAllPicklists, getFields} from '../api';
 import CheckboxMultiSelect from '../components/CheckboxMultiSelect';
 import OrderableTable from '../components/OrderableTable';
+import ValidationFeedback from '../components/ValidationFeedback';
+
+import '../../css/Filters.scss';
+import '../../css/FDSEntries.scss';
```

Comment:
Why do we need to import these styles? It isn't super intuitive why we would import it here

Reply:
That's because we need FDSEntries.scss for the classes to override dropdown item list behavior, as Fields.tsx does. I held a discussion with Clay team about that and they said we probably need to do a feature request to make dropdown match our actual needs.

Reply:
I agree with Bryce here, the scoping here does no look right. If the styles are scoped to a component, they should not be imported in another component. You probably want to do one of two things:
1. Duplicate styles, if there are not a lot of them. This is not such a bad thing, as it is not uncommon that styles of different component drift over time.
2. Extract styles to the smallest common scope. In this case, this can be either a new component style file, something like `Dropdown.scss`, or a global `main.scss`.

Reply:
Done. I've split the styles into one file for component, also removing styles out of scope from FDSEntries.scss